test-utils/src/dev/mod.rs

```diff
@@ -17,6 +17,35 @@ use dropshot::ConfigLogging;
 use dropshot::ConfigLoggingIfExists;
 use dropshot::ConfigLoggingLevel;
 use slog::Logger;
+use std::path::Path;
+
+/// Path to the "seed" CockroachDB directory.
+///
+/// Populating CockroachDB unfortunately isn't free - creation of
+/// tables, indices, and users takes several seconds to complete.
+///
+/// By creating a "seed" version of the database, we can cut down
+/// on the time spent performing this operation. Instead, we opt
+/// to copy the database from this seed location.
+pub const SEED_DB_DIR: &str = "/tmp/crdb-base";
```

Comment:
It'd be good to make this use one of the crates that finds a temp dir for you.  I think elsewhere we do this, and it winds up using `$TMPDIR` from the environment in at least some cases.

Context: /tmp on Helios is generally tmpfs, which comes from swap.  This uses up physical memory.  We already consume a lot of that during builds and tests and it's led to exhaustion even on decently large systems.  There's no need for this to be in-memory.  Plus, allowing people to override this in a standard way allows folks to put this on a ZFS dataset that they've configured differently.

Reply:
Updated to use https://doc.rust-lang.org/std/env/fn.temp_dir.html